### PR TITLE
[Docs] translate 0_config.md into Chinese

### PR DESCRIPTION
**Pull request by wang11wang**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

**Before PR**:

- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects, like MMDet or MMSeg.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: docs/zh_cn/tutorials/0_config.md [hunk 1]
```diff
@@ -1,189 +1,189 @@
...
 - `resnet50`
-- `vit`（will be used in mocov3）
+- `vit`（将会用在mocov3中）
 
-Or there are some special settings which is needed to be mentioned in the config name. E.g:
-- `resnet50-nofrz`: In some downstream tasks，the backbone will not froze stages while training
-- `simclr_resnet50_mocov2-neck`: In simclr, using mocov2-neck to train, not simclr its own neck
+或者其他一些需要在配置名字中强调的特殊的设置。例如：
+- `resnet50-nofrz`：在一些下游任务的训练中，该 backbone 不会冻结 stages
+- `simclr_resnet50_mocov2-neck`：在 simclr 中，使用 mocov2-neck 训练，而不是 simclr 自己的 neck
```
Comment (fangyixiao18) [inline docs/zh_cn/tutorials/0_config.md]: Also remove this line, thanks!

File: docs/zh_cn/tutorials/0_config.md [hunk 2]
```diff
@@ -1,189 +1,189 @@
...
 - `npid-ensure-neg`
 - `deepcluster-sobel`
 
-### Module information
+### 模块信息
 ```
 {backbone setting}_{neck setting}_{head_setting}
 ```
-Here we use `'_'` to concatenate to make the name more readable.
+这里我们使用 `'_'` 连接各个部分提升名字可读性。
```
Comment (fangyixiao18) [inline docs/zh_cn/tutorials/0_config.md]: Remove this line. Actually we forgot to revise this part, now we prefer the format like `{backbone setting}-{neck setting}-{head_setting}`, and the `_` is used like line 30.

Comment (fangyixiao18) [reply docs/zh_cn/tutorials/0_config.md]: Could you please help us to revise the description in English version tutorial? Thanks!

**Review by fangyixiao18 — CHANGES_REQUESTED** [review]

File: docs/zh_cn/tutorials/0_config.md [hunk 3]
```diff
@@ -1,189 +1,187 @@
...
 
-`misc` offers some other algorithm related information. E.g.
+`misc` 提供一些其他算法相关信息。例如：
 - `npid-ensure-neg`
 - `deepcluster-sobel`
 
-### Module information
+### 模块信息
 ```
 {backbone setting}_{neck setting}_{head_setting}
```
Comment (fangyixiao18) [inline docs/zh_cn/tutorials/0_config.md]: Please change this to `{backbone setting}-{neck setting}-{head_setting}`,  and the corresponding line in English version.

File: docs/zh_cn/tutorials/0_config.md [hunk 4]
```diff
@@ -1,189 +1,187 @@
...
-  - `in1k`: Data information, train on ImageNet1k dataset
+  - `mocov2`：算法信息
+  - `resnet50_simclr-neck`：模块信息
+    - `resnet50`：Backbone
+    - `simclr-neck`：使用特殊的 neck，而不是默认的 mocov2-neck
+  - `8xb32-coslr-200e`：训练信息
+    - `8xb32`：共使用 8 张 GPU，每张 GPU上 的 batch size 是 32
+    - `coslr`：使用余弦学习率调度器
+    - `200e`：训练模型 200 个周期
+  - `in1k`：数据信息，在 ImageNet1k 数据集上训练
```
Comment (fangyixiao18) [inline docs/zh_cn/tutorials/0_config.md]: Remove  line 86 - 94, and the corresponding part in English version.

**Review by fangyixiao18 — CHANGES_REQUESTED** [review]
Please revise the two minor parts, the other content looks good. Thanks!

**Review by fangyixiao18 — APPROVED** [review]

**Review by Jiahao000 — APPROVED** [review]

**Comment by OpenMMLab-Assistant003** [comment]
Hi @wang11wang！First of all, we want to express our gratitude for your significant PR in the MMSelfsup project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA

If you have WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤ @wang11wang

**Pull request merged**